backend/ml/management/commands/ml_train.py

```diff
@@ -117,7 +141,12 @@ def save_data(video_scores, contributor_rating_scores):
     rating_ids.update(
         {(rating.user_id, rating.entity_id): rating.id for rating in created_ratings}
     )
-    ContributorRatingCriteriaScore.objects.all().delete()
+
+    if trusted_only:
+        ContributorRatingCriteriaScore.objects.filter(contributor_rating__user__in=User.trusted_users()).delete()
+    else:
+        ContributorRatingCriteriaScore.objects.exclude(contributor_rating__user__in=User.trusted_users()).delete()
+
```

Comment:
Only delete the one that will be re-generated. All trusted user in trusted mode and only non trusted user in all mode